### PR TITLE
fix tests for FF 44

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @churkin @LavrovArtem

### CODE REVIEW

**Comment by churkin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

File: test/client/data/cross-domain/service-worker-not-available.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 <script>
     var postMessage = hammerhead.sandbox.event.message.postMessage;
 
-    postMessage(window.top, [!hammerhead.nativeMethods.registerServiceWorker ? 'success' : 'fail', '*']);
+    postMessage(window.top, [hammerhead.nativeMethods.registerServiceWorker ? true : false, '*']);
```
Comment (inikulin) [inline test/client/data/cross-domain/service-worker-not-available.html]: Why not just:
```js
postMessage(window.top, [!!hammerhead.nativeMethods.registerServiceWorker, '*']);
```

**Comment by inikulin** [comment]
\r-

**Pull request merged**